meteor/sofie-server/data-access/repositories/test/mongo-piece-repository.spec.ts

```diff
@@ -88,12 +94,12 @@ describe(`${MongoPieceRepository.name}`, () => {
 	})
 
 	// TODO: Extract to Helper Class in Model layer
-	function createPiece(params: { id?: string; name?: string; rank?: number; partId?: string }): Piece {
-		return new Piece({
-			id: params.id ?? testDatabase.getValidObjectIdString('id'),
+	function createPiece(params: { id?: string; name?: string; partId?: string }): MongoPiece {
```

Comment:
You are creating a `MongoPiece`, not a `Piece`, so the method should be named `createMongoPiece`

Reply:
Has been renamed.